src/sentry/utils/json.py

```diff
@@ -142,6 +143,29 @@ def loads(
             return _default_decoder.decode(value)
 
 
+# loads JSON with `orjson` or the default function depending on `option_name`
+# TODO: remove this once we're confident that orjson is working as expected
+def loads_experimental(option_name: str, data: str | bytes, skip_trace: bool = False) -> JSONData:
+    from sentry.features.rollout import in_random_rollout
+
+    if in_random_rollout(option_name):
+        with sentry_sdk.start_span(op="sentry.utils.json.loads") if not skip_trace else nullcontext():  # type: ignore[attr-defined]
```

Comment:
this should not be type ignored: https://www.notion.so/sentry/sentry-typing-guidelines-best-practices-471d5aacf262476188982ce70f909e00?pvs=4#4c5f66d5e5f2460991cf1074111fdd85

Reply:
Unfortunately removing the comment doesn't work in this case—is there something I can do to make this type check? The error I get is
```python
src/sentry/utils/json.py:139: error: "object" has no attribute "__enter__"  [attr-defined]
src/sentry/utils/json.py:139: error: "object" has no attribute "__exit__"  [attr-defined]
```

Reply:
you'll need to teach the type checker that this is a context manager since it is resolving `Union[nullcontext, ContextManager[Whatever]]` as `object`:

```
if skip_trace:
    ctx: ContextManager[object] = nullcontext()
else:
    ctx = ...
```

or what I prefer:

```python
with contextlib.ExitStack() as ctx:
    if not skip_trace:
        ctx.enter_context(...)
    ...
```